main.py

```diff
@@ -214,7 +212,12 @@ def main(collect: bool = False, resume: Optional[Path] = None):
         parameters['uploads'] = FileManager.file_paths_to_dict(resume, plain_text_resume_file)
         parameters['outputFileDirectory'] = output_folder
         parameters['collectMode'] = collect
-        
+        # check if the config is not set as well
+        BrowserFactory.set_browser_type(
+            BrowserType.CHROME if BROWSER_TYPE_CONFIG is None or BROWSER_TYPE_CONFIG == BrowserType.CHROME
```

Comment:
move this to method - line 217, 218

Reply:
fixed, replaced with `BrowserFactory.init_browser_type()` that read the config file inside the factory